pallets/rc-migrator/src/crowdloan.rs

```diff
@@ -213,6 +214,16 @@ impl<T: Config> PalletMigration for CrowdloanMigrator<T>
 					let mut contributions_iter = pallet_crowdloan::Pallet::<T>::contribution_iterator(fund.fund_index);
 
 					while let Some((contributor, (amount, memo))) = contributions_iter.next() {
+						if weight_counter.try_consume(T::DbWeight::get().reads_writes(1, 1)).is_err() {
```

Comment:
I would rather use `consume` here, but it might panic bc of `debug_assert`.

Reply:
Hm, yes looks like we do not have a saturating consume.